src/helpers/files.js

```diff
@@ -160,9 +194,10 @@ const FilesWorkspacePlugin = {
 		if (fileList.id !== 'files' && fileList.id !== 'files.public') {
 			return
 		}
-
+		addMenuRichWorkspace()
```

Comment:
I'd say we can just move the `menu.addMenuEntry` call in here and avoid the two plugins being nested to just have one for the rich workspace.